utils/utils.validation.js

```diff
@@ -0,0 +1,48 @@
+exports.validateArticleId = (article_id) => {
+  const articleId = Number(article_id);
+  if (isNaN(articleId)) {
+    return Promise.reject({
+      status: 400,
+      msg: "400 - Bad Request: Invalid article_id",
+    });
+  }
+  return Promise.resolve(articleId);
+};
+
+exports.validateUsername = (username) => {
+  if (username === undefined) {
+    return Promise.reject({
+      status: 400,
+      msg: "400 - Bad Request: Missing username or body in request body",
+    });
```

Comment:
The comments table should do this for you. You should handle this with an SQL error.